### ISSUE TITLE
ARM: JIT Regression tests b26496 and b539509 are failing in Release mode

### ISSUE THREAD

**Issue by prajwal-aithal** [dotnet/coreclr#6289]
We recently enabled cross building and running tests for the Linux ARM Emulator (process documented here: #6249 ) and added this to the per-PR CI check process ([Debug](http://dotnet-ci.cloudapp.net/job/dotnet_coreclr/job/master/job/arm_emulator_cross_debug_ubuntu_prtest/) and [Release](http://dotnet-ci.cloudapp.net/job/dotnet_coreclr/job/master/job/arm_emulator_cross_release_ubuntu_prtest/) jobs)

But the following tests are failing for the Release mode (succeed in the Debug mode) as can be seen [here](http://dotnet-ci.cloudapp.net/job/dotnet_coreclr/job/master/job/arm_emulator_cross_release_ubuntu_prtest/494/console).
* `JIT/Regression/VS-ia64-JIT/V1.2-M02/b26496/b26496/b26496.sh`
* `JIT/Regression/VS-ia64-JIT/V2.0-RTM/b539509/b539509/b539509.sh`

Can these failures be investigated and resolved?

@jyoungyun @sjsinju @chunseoklee @parjong @leemgs

**Comment by wateret** [comment]
As the error message is:
```
16:55:35                Unhandled Exception: 
16:55:35                   Cannot print exception string because Exception.ToString() failed.
```

This probably is the same issue with #6227.

**Comment by RussKeldorph** [comment]
@prajwal-aithal @CarolEidt Should this issue be closed?

**Comment by CarolEidt** [comment]
Yes, I believe this should be closed with #6227 which reverted #6021.
A revised version of the original changes (#6579) has been submitted, and which passes these tests.
